lib/validators/and.js

```diff
@@ -13,6 +13,7 @@ class And extends Validator {
   }
 
   async validate (context, validationSettings, registry) {
+    this.logUsage(context, validationSettings)
```

Comment:
Instead of repetitively adding this one line in every single validator, I'd recommend adding a method in the super class `Validator.processValidation` and have `flex` call that instead. Within `processValidation` you'd have two lines of code:

```js
   this.logUsage(context, validationSettings)
   this.validate(context, validationSettings, registry)
```

This way logUsage is always called for all validators and when implementing additional validators you ensure it is logged.

Reply:
that's a great idea, i'll implement it

Reply:
done implementing and tests also fixed